homeassistant/components/rainmachine/strings.json

```diff
@@ -11,11 +11,11 @@
       }
     },
     "error": {
-      "identifier_exists": "Account already registered",
-      "invalid_credentials": "Invalid credentials"
+      "identifier_exists": "[%key:common::config_flow::abort::already_configured_account%]",
```

Comment:
Is use of an "abort" key in "error" section is OK ?
Should i rename the key also ?

Reply:
Yes to both questions.

Reply:
@springstan just to be sure : rename it to  `already_configured_account` ?

Reply:
No `already_configured` is fine.

Reply:
@balloob merged before i can do it :scream: